CHANGELOG.md

```diff
@@ -9,6 +9,32 @@ Unreleased section should follow [Release Toolkit](https://github.com/newrelic/r
 
 ## Unreleased
 
+### enhancement
+- Add metrics collection from DockerAPI to support nodes with cgroup V2 on ECS clusters.
+
+### security
+- Updated logrus dependency to remediate vulnerability
+
+### breaking
+- Memory swap metrics `memorySwapUsageBytes`, `memorySwapOnlyUsageBytes` and `memorySwapLimitUsagePercent` will not be sent if not present. Before they were being reported with value `0` (#224)
```

Comment:
It's going to happen the same for the [blkio](https://github.com/newrelic/nri-docker/blob/NR-221180-docker-api-fetcher/src/nri/sampler.go#L274) metrics isn't it?

Reply:
thanks i added the note